lib/pavilion/test_run.py

```diff
@@ -988,8 +988,13 @@ def __repr__(self):
         return "TestRun({s.name}-{s.id})".format(s=self)
 
     def _get_skipped(self):
-        skip_reason_list = self._evaluate_skip_conditions()
-        matches = " ".join(skip_reason_list)
+        """Kicks off assessing if current test is skipped."""
+        if self.status.current().state == 'SKIPPED':
```

Comment:
You shouldn't rely on status like this. Add 'skipped' as a test attribute (there's a new 'attributes' file).

Reply:
See my comment on mattermost.